portfolio/src/main/webapp/script.js

```diff
@@ -29,14 +29,34 @@ function addRandomFact() {
 
 
 function getFromServlet(){
-    fetch('/data').then(response => response.text()).then((saying) => { 
+    const param = new URLSearchParams(window.location.search);
+    var maxParameter = param.get('load-comments');
+    if (maxParameter == null){
+        maxParameter = 5;
+    }
 
-    const sayingContainer = document.getElementById('saying-container');
-    console.assert(sayingContainer, "You have no element with ID 'saying-container'"); 
-    sayingContainer.innerHTML = saying;
+    fetch('/data?load-comments='+maxParameter).then(response => response.text()).then((comment) => { 
     
-    })
+      const commentContainer = document.getElementById('comment-container');
+      console.assert(commentContainer, "You have no element with ID 'comment-container'"); 
+
+      const list = document.getElementById('comment-container');
+      comment = comment.replace("[", "");
+      comment = comment.replace("]","");
+      var commentArray = comment.split(',');
+      for(i =0;  i < commentArray.length; i++){
+          list.appendChild(createListElement(commentArray[i]));
+        }
+    });
```

Comment:
It looks like some of this code is already under discussion in #14 . Please resolve those comments and merge those in and then ask for further review here.